document-store/src/main/java/org/hypertrace/core/documentstore/TypesafeDatastoreConfigAdapter.java

```diff
@@ -32,7 +33,8 @@ public DatastoreConfig convert(final Config config) {
               null,
               null,
               AggregatePipelineMode.DEFAULT_ALWAYS,
-              DataFreshness.SYSTEM_DEFAULT) {
+              DataFreshness.SYSTEM_DEFAULT,
+              Duration.ofMinutes(20)) {
```

Comment:
What does 20 mins mean here?

Reply:
Default query timeout in case no timeout is specified by the clients.